src/async/__tests__/index.browser.js

```diff
@@ -0,0 +1,14 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+/* eslint-env node */
+import './__browser__/prepare-render.browser.js';
+
+process.on('unhandledRejection', e => {
```

Comment:
Weird, why do we need this file? I don't think we have it for other tests, right?

Reply:
oops, bad copy paste